lib/sw_indexer_engine.rb

```diff
@@ -32,4 +35,9 @@ def index(druid, targets = nil)
   def purl_model(druid)
     DiscoveryIndexer::InputXml::Purlxml.new(druid).load
   end
+
+  def update_marc_record(druid)
```

Comment:
Sure that's possible, but I guess I don't see a lot of pragmatic value in this.  The code calling these methods is only being executed a few lines above and the overall class itself is already pretty short (the new class would be only a few lines).  Based on how the tests are written, we'd have to alter the test setup and some expectations as well.